numeral.js

```diff
@@ -249,6 +259,7 @@
             abbrB = false, // force abbreviation to billions
             abbrT = false, // force abbreviation to trillions
             abbrForce = false, // force abbreviation
+            omitAbbr = false,
```

Comment:
FWIW, I believe my change doesn't handle the case of including `~` in a format that doesn't include any kind of abbreviation or % sign. But this is an invalid format that makes no sense, so I think it's okay.